privacy-snapshot.json

```diff
@@ -87,5 +87,7 @@
   "unresponsive-rpc.url",
   "user-storage.api.cx.metamask.io",
   "verify.walletconnect.com",
-  "www.4byte.directory"
+  "www.4byte.directory",
+  "i.seadn.io",
+  "images.ctfassets.net"
```

Comment:
New mocks for images we load during tests.
These are mock notification nft images and announcement assets.